<launch>
  <node pkg="rubot_projects" type="project.py" name="movebase_client_waypoints" output="screen" >
    <rosparam file="$(find rubot_projects)/config/goals_projecct_final.yaml" command="load" />
  </node>
</launch>